pkg/obfuscate/obfuscate.go

```diff
@@ -103,6 +103,9 @@ type StatsClient interface {
 // SQLConfig holds the config for obfuscating SQL.
 // easyjson:json
 type SQLConfig struct {
+	// DBMS identifies the type of database management system (e.g. MySQL, Postgres, and SQL Server).
```

Comment:
```suggestion
	// DBMS identifies the type of database management system (e.g. MySQL, Postgres, and SQL Server).
	// Valid values for this can be found at https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/database.md#connection-level-attributes
```